sky/templates/oci-ray.yml.j2

```diff
@@ -85,14 +89,20 @@ setup_commands:
   # Line 'sudo grep ..': set the number of threads per process to unlimited to avoid ray job submit stucking issue when the number of running ray jobs increase.
   # Line 'mkdir -p ..': disable host key check
   # Line 'python3 -c ..': patch the buggy ray files and enable `-o allow_other` option for `goofys`
-  - sudo systemctl stop unattended-upgrades || true;
+  - echo "setup commands runs at $(date)" > /tmp/provision.tmp.out || true;
+    {%- if os_type == "ubuntu" %}
```

Comment:
Do we really need to pass `os_type` all along the image catalog? I am wondering if we can just check the os type using `uname` or other commands.

Reply:
Thanks @Michaelvll for the comments.  The main issue here is for the SSH user name. We need to specify different username for different OS type. "ubuntu" for Ubuntu OS, "opc" for other OS like "oraclelinux".

Reply:
I see! That makes sense.